### ISSUE TITLE
add support for opening all diffs relative to a specific earlier patchset

### ISSUE THREAD

**Issue by fischman** [ascherkus/reviewtool#1]
Filing as an issue because I hear patches are NOT welcome!

**Comment by ascherkus** [comment]
patches are INDEED VERY WELCOME

**Comment by ascherkus** [comment]
I too thought about this but riddle me this: what is the expected behaviour when there isn't a diff for all files? skip it?

**Comment by fischman** [comment]
Yes, skip it files with no diffs, since they are irrelevant to the followup
review.
EXCEPT, if there were comments on a given file in the previous patchset and
there is no update file in the new patchset; in that case open the old
patchset against base to make clear to the reviewer that the reviewee
didn't address comments with code changes.
